packages/twenty-server/src/engine/metadata-modules/agent/agent-handoff.entity.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  Column,
+  CreateDateColumn,
+  DeleteDateColumn,
+  Entity,
+  Index,
+  JoinColumn,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+  Unique,
+  UpdateDateColumn,
+} from 'typeorm';
+
+import { Relation } from 'src/engine/workspace-manager/workspace-sync-metadata/interfaces/relation.interface';
+
+import { Workspace } from 'src/engine/core-modules/workspace/workspace.entity';
+
+import { AgentEntity } from './agent.entity';
+
+@Entity('agent_handoff')
+@Index('IDX_AGENT_HANDOFF_ID_DELETED_AT', ['id', 'deletedAt'])
+@Unique('IDX_AGENT_HANDOFF_FROM_TO_WORKSPACE_UNIQUE', [
+  'fromAgentId',
+  'toAgentId',
+  'workspaceId',
+])
```

Comment:
Since we have soft delete on this table then maybe this needs to be a partial index. See UQ_USER_EMAIL for example.

Could you please then create an issue to look at all other core entities that have a deletedAt field + @Unique but no partial index? It seems like this is a problem. But it's not worth fixing or discussing it in this PR. If you don't mind opening another PR that'd be great. Thanks!